index.html

```diff
@@ -94,22 +94,23 @@
 
       <div class="col-md-9 col-12" id="content"></div>
       <!-- Button examples will go here using JS -->
+      <div>
+        <a class="buttonload" href="#"><i class="fa fa-spinner fa-spin fa-1x fa-fw"></i>
+          <span class="sr-only">Loading...</span> LOADING</a>
+        <a class="buttonload" href="#"><i class="fa fa-circle-o-notch fa-spin fa-1x fa-fw"></i>
+          <span class="sr-only">Loading...</span> LOADING</a>
+        <a class="buttonload" href="#"><i class="fa fa-refresh fa-spin fa-1x fa-fw"></i>
+          <span class="sr-only">Loading...</span> LOADING</a>
+        <a class="buttonload" href="#"><i class="fa fa-cog fa-spin fa-1x fa-fw"></i>
+          <span class="sr-only">Loading...</span> LOADING</a>
+        <a class="buttonload" href="#"><i class="fa fa-spinner fa-pulse fa-1x fa-fw"></i>
+          <span class="sr-only">Loading...</span> LOADING</a>
+      </div>
```

Comment:
You don't need to add the html here just take look at the [buttons-examples.js](https://github.com/shahednasser/sbuttons/blob/master/assets/js/buttons-examples.js) and add it to the buttons object with required propeties.